frontend/src/components/dashboard/TripsSection/TripsSection.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import styles from "./TripsSection.module.css";
+import { CustomCircularProgress } from "./TripsSection.styles";
+import TripsList from "../commons/TripsList/TripsList";
+
+export default function GroupsSection({ title, trips, loading, disable }) {
+    return (
+        <div className={styles.section}>
+            <h1>{title}</h1>
+            {loading && <CustomCircularProgress />}
```

Comment:
I'm wondering if we can make the spinner a bit smaller on the screen - just a thought for in future, could be a separate issue